utils/httputils/httputils.go

```diff
@@ -0,0 +1,6 @@
+package httputils
```

Comment:
what do you think about renaming the file hierarchy to: `utils/http/utils.go` keeping the package name as `httputils`?